spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/trace/StackdriverTraceAutoConfigurationTests.java

```diff
@@ -66,50 +63,19 @@ public void test() {
 			SleuthProperties sleuthProperties = context.getBean(SleuthProperties.class);
 			assertThat(sleuthProperties.isTraceId128()).isTrue();
 			assertThat(sleuthProperties.isSupportsJoin()).isFalse();
-
```

Comment:
I get that a lot of things were deleted, and a lot of the responsibility has shifted from this module to the dependency. But would it still be good to have some tests that see if the things are still working as they should ? (perhaps that would be more of an integration test).

Reply:
I wanted Trace integration tests too. I looked a while back into implementing them, but found out the client library won't let you query traces by a key, which makes the integration tests infeasible..
I did test this manually, though, and it works.